OlinkAnalyze/R/detect_data_type.R

```diff
@@ -0,0 +1,31 @@
+#' Help function checking whether the input data contains
+#' a column named "NPX" or "Quantified_value".
+#'
+#' @author
+#'  Simon Forsberg
+#'  Masoumeh Sheikhi
+#' @param df An arrow object
+#' @return A character string indicating the detected data type
+#' ("NPX" or "Quantified_value").
+#'
+detect_data_type <- function(df) {
```

Comment:
This function will change a lot because we it will be using the dictionary to identify the quantification method, however, for the time being it is doing the job.

In any case we should also allow Ct data in addition to Quantified and NPX.